grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy

```diff
@@ -492,6 +499,16 @@ class GrailsGradlePlugin extends GroovyPlugin {
         }
     }
 
+    protected void configureRunScript(Project project) {
+        project.tasks.create("runScript", ApplicationContextScriptTask) {
+            classpath = project.sourceSets.main.runtimeClasspath + project.configurations.console
+            systemProperty 'grails.env', System.getProperty("grails.env", "development")
```

Comment:
Maybe use the constants defined here for these https://github.com/grails/grails-core/blob/master/grails-bootstrap/src/main/groovy/grails/util/Environment.java#L65